solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -72,6 +73,7 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
 
     ILiquidityProtectionSettings public immutable settings;
     ILiquidityProtectionStore public immutable store;
+    ILiquidityProtectionStats public immutable stats;
```

Comment:
can use `using SafeERC20 for IERC20;` with OZ's SafeERC20, cleaner

Reply:
Changing to from `TokenHandler` to `SafeERC20` requires changing from `IERC20Token` to `IERC20`, which in turn requires changes all over the place (LP contract, LP test contract, LP interface, `DSToken` contract, `DSToken` interface and probably any other contract or interface which relies on this interface).
We shouldn't apply this fix as part of the migration task, but in a separate task (and on a separate branch).